packages/expo-constants/ios/ConstantsModule.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ConstantsModule.swift
+//  ExpoConstants-ExpoConstants
+//
+//  Created by Alan Hughes on 11/11/2022.
+//
+
+import ExpoModulesCore
+import WebKit
+
+public class ConstantsModule: Module {
+
+    private lazy var constants = {
+        let dict = appContext?.constants?.constants()
+
+        if let result = dict as? [String: Any?] {
+            return result
+        }
+
+        return [String: Any?]()
+    }()
```

Comment:
Probably it can be simplified to:

```suggestion
    private lazy var constants = appContext?.constants?.constants() as? [String: Any] ?? [:]
```